src/components/ExchangePage/index.tsx

```diff
@@ -1304,11 +1266,7 @@ function ExchangePage({ sendingInput = false, history, params }: ExchangePagePro
                     warningMedium={warningMedium}
                     warningHigh={warningHigh}
                   >
-                    {priceSlippage
-                      ? priceSlippage.toFixed(4) === '0.0000'
-                        ? '<0.0001%'
-                        : priceSlippage.toFixed(4) + '%'
-                      : '-'}
+                    {priceSlippage?.toFixed(2) === '0.00' ? '<0.01%' : `${priceSlippage?.toFixed(2)}%` ?? '-'}
```

Comment:
weird expression, wouldn't 0.009 round to 0.01 or does it truncate? prefer .lessThan(Percent(1,100))

Reply:
yeah it would round to 0.01 by default (thought his could be changed by passing additional arguments to toFixed). i was just replicating existing logic but agreed its a bad check, fixed